### DISCUSSION TITLE
RustDesk capability ?

### DISCUSSION THREAD

**Discussion by bluray91** [rustdesk/rustdesk#1400] [Q&A] [answered]
Hello,

I find this tool very promising and being self hostable and free of use, very interesting.

But I can't seem to find the following information:

> How much devices can be handled by RustDesk ?

_I'm working in a company where there's ~6000 users and I would like to know if RustDesk could handle them._

Or is RustDesk dedicated for smaller size infrastructure ?

Thanks in advance !

**Comment by rustdesk** [comment, marked as answer]
Our pro version can, but it is not ready to publish yet, our all resources are put on flutter desktop rewriting for now.

**Reply by bluray91** [reply]
Got it, thanks a lot for your response.

**Reply by smartekIT** [reply]
curious to know what's the limit on the current published version?
